arch/arm/core/aarch32/mpu/arm_mpu.c

```diff
@@ -32,6 +34,13 @@ LOG_MODULE_DECLARE(mpu);
 #define NUM_MPU_REGIONS   DT_PROP(MPU_NODEID, arm_num_mpu_regions)
 #endif
 
+#define DT_NODE_HAS_PROP_AND_OR(node_id, prop) \
```

Comment:
This is use of the reserved DT namespace and needs to be renamed; I suggest NODE_HAS_PROP_OR or something like that.

Reply:
Fair enough, I'll rename it.

Reply:
done